lib/gui/components/drive-selector/controllers/drive-selector.js

```diff
@@ -90,35 +97,18 @@ module.exports = function (
       return $q.resolve(false)
     }
 
-    const isLargeDrive = constraints.isDriveSizeLarge(drive)
-    const isRecommendedDrive = constraints.isDriveSizeRecommended(drive, selectionState.getImage())
-
-    if (isLargeDrive || !isRecommendedDrive) {
-      const messageTuples = [
-        [
-          isLargeDrive,
-          messages.warning.largeDriveSize({
-            humanDriveSize: units.bytesToClosestUnit(drive.size)
-          })
-        ],
-        [
-          !isRecommendedDrive,
-          messages.warning.unrecommendedDriveSize({
-            drive,
-            image: selectionState.getImage()
-          })
-        ]
-      ]
-
-      const concatenatedMessage = _.reduce(messageTuples, (accumulatedMessage, messageTuple) => {
-          if (messageTuple[0]) {
-            return `${accumulatedMessage}\n\n${messageTuple[1]}`
-          } else {
-            return accumulatedMessage
-          }
-        }, '')
+    if (constraints.isDriveSizeLarge(drive)) {
+      return confirmationWarningModal(messages.warning.largeDriveSize({
+        humanDriveSize: units.bytesToClosestUnit(drive.size),
```

Comment:
Hmmm, borrowing the 'pattern' from the display-units PR, I wonder if it _might_ make sense for Etcher to automatically add a `humanSize` property to every drivelist object, to prevent this `units.bytesToClosestUnit` being done multiple times in different places?